downstream/modules/platform/proc-import-mesh-ca.adoc

```diff
@@ -0,0 +1,29 @@
+[id="proc-import-mesh-ca_{context}"]
+
+= Importing a Certificate Authority (CA) certificate
+A Certificate Authority (CA) is used to certify and sign execution nodes in an {AutomationMesh} environment. Users who wish to provide their own CA can do so by specifying the path to the certificate and the private RSA key file in the `inventory` file of their {PlatformName} installer.
+
+NOTE: Users are not required to have a CA prior to their {AutomationMesh} installation. Users without a CA can proceed with their {PlatformNameShort} installation where the installer will generate a CA for the user using OpenSSL.
+
+.Prerequisites
+To import your own CA certificate, it must meet the following requirements:
+
+* asdasd
+* asdasd
+
+.Procedure
+* In the `inventory` file, add the following variables and specify a path to both the certificate file (`.crt`) and the private RSA key (`.key`), following the example below:
```

Comment:
I feel like this could be broken into steps.

. Open the `inventory` file for editing.
. Add the `mesh_ca_keyfile` variable and specify the full path to the CA certificate file (`.crt`).
. Add the `mesh_ca_certfile` variable and specify the full path to the private RSA key (`.key`).
. Save the changes to the `inventory` file.

Reply:
This is a much elegant way to structure this. Thanks Don!